browser/src/Renderer/WebGL/WebGLTextRenderer.ts

```diff
@@ -9,16 +9,17 @@ import {
 
 // tslint:disable-next-line:no-bitwise
 const maxGlyphInstances = 1 << 14 // TODO find a reasonable way of determining this
-const glyphInstanceFieldCount = 12
+const glyphInstanceFieldCount = 13
 const glyphInstanceSizeInBytes = glyphInstanceFieldCount * Float32Array.BYTES_PER_ELEMENT
 
 const vertexShaderAttributes = {
     unitQuadVertex: 0,
     targetOrigin: 1,
     targetSize: 2,
     textColorRGBA: 3,
-    atlasOrigin: 4,
-    atlasSize: 5,
+    atlasLayerIndex: 4,
```

Comment:
Really cool that we can index into the atlas layer from this vertex data; thanks for plumbing that through! 👍 Neat to see how it works with the `sampler2DArray` and how the lookup works in fragment shader.